gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/util/policy/ServicePolicyXMLSimplifier.java

```diff
@@ -0,0 +1,86 @@
+package com.ca.apim.gateway.cagatewayexport.util.policy;
+
+import com.ca.apim.gateway.cagatewayconfig.beans.Service;
+import com.ca.apim.gateway.cagatewayconfig.bundle.loader.ServiceAndPolicyLoaderUtil;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentParseException;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.text.StringEscapeUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.inject.Singleton;
+import java.util.Base64;
+import java.util.logging.Logger;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.properties.PropertyConstants.L7_TEMPLATE;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleChildElement;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleElement;
+
+
+@Singleton
+public class ServicePolicyXMLSimplifier {
+    private static final Logger LOGGER = Logger.getLogger(ServicePolicyXMLSimplifier.class.getName());
+    /**
+     *
+     * @param policyElement service policy element.
+     * @param service Service
+     */
+    public void simplifyServicePolicyXML(Element policyElement, Service service) {
+        service.getProperties().putIfAbsent(L7_TEMPLATE, "false");
+        // [Set as Portal Managed Service] assertion
+        simplifyPortalManagedAssertion(policyElement, service);
+    }
+
+    private void simplifyPortalManagedAssertion(Element policyElement, Service service) {
+        try {
+            Element portalManagedElement = getSingleElement(policyElement, API_PORTAL_INTEGRATION);
+            Element flagElement = getSingleChildElement(portalManagedElement, API_PORTAL_INTEGRATION_FLAG, true);
+
+            if (flagElement != null && API_PORTAL_INTEGRATION_FLAG_SERVICE.equals(flagElement.getAttribute(STRING_VALUE))) {
+                Element enabledElement = getSingleChildElement(portalManagedElement, ENABLED, true);
+                Element variablePrefixElement = getSingleChildElement(portalManagedElement, API_PORTAL_INTEGRATION_VARIABLE_PREFIX, true);
+                Element apiIdElement = getSingleChildElement(portalManagedElement, API_PORTAL_INTEGRATION_API_ID, true);
+                Element apiGroupElement = getSingleChildElement(portalManagedElement, API_PORTAL_INTEGRATION_API_GROUP, true);
+                Document document = policyElement.getOwnerDocument();
+                Element parentElement = (Element) portalManagedElement.getParentNode();
+                boolean isEnabled = enabledElement == null || Boolean.parseBoolean(enabledElement.getAttribute(BOOLEAN_VALUE));
+                String variablePrefix = variablePrefixElement != null ? variablePrefixElement.getAttribute(STRING_VALUE) : "portal.managed.service";
+
+                service.getProperties().put(L7_TEMPLATE, Boolean.toString(isEnabled));
```

Comment:
template is the word to be used for encasses,  is it OK to keep this property in service properties

Reply:
It's just metadata. If nobody is using it, no issues. After all, I'm trying to match and capture the existing information.
I believe it might not be useful to the current Portal.